users/views.py

```diff
@@ -41,6 +45,43 @@ def post(self, *args, **kwargs):
             'user_id': serializer.user.id
         }, status=200)
 
+class ChangePassword(APIView):
+    """
+        User Password EndPoint
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def get(self, *args, **kwargs):
+        """
+            Check if User has already set a password
+        """
+        checkflag = False
+        user = self.request.user
+        if user.has_usable_password():
+            checkflag = True
+
+        return Response(checkflag, status=200)
+    
+    def post(self, *args, **kwargs):
+        """
+            Create User Password
+        """
+        serializer = AddPasswordSerializer(
+            data=self.request.data, request=self.request)
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response({'Success': 'Password added'}, status=200)
+
+    def put(self, *args, **kwargs):
+        """
+            Update User Password
+        """
+        serializer = ChangePasswordSerializer(
+            data=self.request.data, request=self.request)
+        
+        serializer.is_valid(raise_exception=True)
+        serializer.save()
+        return Response({'Success':"Update is a Success"}, status=200)
```

Comment:
no need to add string message if it is not needed. We don't declare the error string messages on the backend. we just need a correct status code.

by the way, updated is 204 not 200